### PR TITLE
Add some acceptance testing for Packer init commands

### PR DESCRIPTION
**Pull request by azr**

### CODE REVIEW

File: command/init.go [hunk 1]
```diff
@@ -116,6 +116,7 @@ func (c *InitCommand) RunContext(buildCtx context.Context, cla *InitArgs) int {
 		})
 		if err != nil {
 			c.Ui.Error(err.Error())
+			ret = 1
```
Comment (azr) [inline command/init.go:119]: Tests made me find this one !

File: command/init_test.go [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+			"already-installed-no-op",
+			map[string]string{
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_darwin_amd64":                "1",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_darwin_amd64_SHA256SUM":      "6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4b",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_windows_amd64.exe":           "1.exe",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_windows_amd64.exe_SHA256SUM": "07d8453027192ee0c4120242e6e84e2ca2328b8e0f506e2f818a1a5b82790a0b",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_linux_amd64":                 "1.out",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_linux_amd64_SHA256SUM":       "59031c50e0dfeedfde2b4e9445754804dce3f29e4efa737eead0ca9b4f5b85a5",
+			},
+			"h1:Q5qyAOdD43hL3CquQdVfaHpOYGf0UsZ/+wVA9Ry6cbA=",
```
Comment (azr) [inline command/init_test.go:54]: This is like a directory hash for the whole plugin directory; just to make sure nothing else is added in there.

File: command/init_test.go [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+			},
+			"release-with-no-binary",
+			nil,
+			"h1:47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=",
+			`# cfg.pkr.hcl
+			packer {
+				required_plugins {
+					comment = {
+						source  = "github.com/sylviamoss/comment"
+						version = "v0.2.20"
```
Comment (azr) [inline command/init_test.go:137]: let's make sure to never change this version !

File: command/init_test.go [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_linux_amd64":                 "1.out",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_linux_amd64_SHA256SUM":       "59031c50e0dfeedfde2b4e9445754804dce3f29e4efa737eead0ca9b4f5b85a5",
+			},
+			"h1:Q5qyAOdD43hL3CquQdVfaHpOYGf0UsZ/+wVA9Ry6cbA=",
+			`# cfg.pkr.hcl
+			packer {
+				required_plugins {
+					comment = {
+						source  = "github.com/sylviamoss/comment"
+						version = "v0.2.018"
```
Comment (sylviamoss) [inline command/init_test.go:60]: Why is it  `v0.2.018` and not `v0.2.18`

File: command/init_test.go [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+			[]string{
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_darwin_amd64",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_darwin_amd64_SHA256SUM",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_linux_amd64",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_linux_amd64_SHA256SUM",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_windows_amd64.exe",
+				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_windows_amd64.exe_SHA256SUM",
+				map[string]string{
+					"darwin":  "github.com/sylviamoss/comment/packer-plugin-comment_v0.2.19_x5.0_darwin_amd64_SHA256SUM",
+					"linux":   "github.com/sylviamoss/comment/packer-plugin-comment_v0.2.19_x5.0_linux_amd64_SHA256SUM",
```
Comment (sylviamoss) [inline command/init_test.go:108]: nice!

File: command/init_test.go (hunk 3 above)
Comment (sylviamoss) [reply command/init_test.go:137]: Good call!

File: command/init_test.go (hunk 4 above)
Comment (azr) [reply command/init_test.go:60]: It's intentional, to check that I can mess up the versions a little and that ultimately they match with the one installed that does not have the 0.

Comment (sylviamoss) [reply command/init_test.go:60]: Oh, nice!!

**Pull request merged**